browser/src/control/AutoCompletePopup.ts

```diff
@@ -65,11 +65,12 @@ abstract class AutoCompletePopup {
 	abstract onAdd(): void;
 
 	closePopup(): void {
-		var popupExists = L.DomUtil.get(this.popupId);
+		var popupExists = L.DomUtil.get(this.popupId + 'List');
```

Comment:
@Rash419 After this change, if I type @ followed by something that doesn't mater, e.g. aaaa, then the 'no search results found' popup doesn't disappear on pressing esc